packages/design-toolkit/src/components/drawer/types.ts

```diff
@@ -254,3 +266,7 @@ export type DrawerContextValue = {
   unregister: (view: UniqueId) => void;
   placement: XAxisUnion | YAxisUnion;
 };
+
+export type DrawerCloseProps = {
+  id?: UniqueId;
```

Comment:
I think this should be `for` to maintain consistency with other components like `DrawerMenuItem` and `DrawerTrigger` 
```suggestion
  for?: UniqueId;
```